i18n/id.json

```diff
@@ -0,0 +1,65 @@
+{
+	"@metadata": {
+		"authors": [
+			"Rumanti"
+		]
+	},
+	"skinname-scratchwikiskin2": "ScratchWikiSkin",
+	"scratchwikiskin-desc": "{{desc|what=skin|name=ScratchWikiSkin2|url=https://github.com/InternationalScratchWiki/ScratchWikiSkin2}}",
+	"scratchwikiskin-create": "Buat",
+	"scratchwikiskin-create-project": "Buat karya baru",
+	"scratchwikiskin-create-page": "Buat halaman baru",
+	"scratchwikiskin-create-page-url": "Scratch-Indo-Wiki:Buat_sebuah_artikel",
+	"scratchwikiskin-explore": "Jelajah",
+	"scratchwikiskin-explore-projects": "Jelajah karya",
+	"scratchwikiskin-discuss": "Bahas",
+	"scratchwikiskin-discuss-text": "Bahas Scratch",
+	"scratchwikiskin-discuss-wiki": "https:\/\/scratch.mit.edu\/discuss\/topic\/284040\/",
+	"scratchwikiskin-discuss-wiki-text": "Topik forum wiki",
+	"scratchwikiskin-tips": "Tips",
+	"scratchwikiskin-tips-text": "Tips Scratch",
+	"scratchwikiskin-faq-page-url": "Pertanyaan_Sering_Diajukan",
+	"scratchwikiskin-faq-page": "Pertanyaan yang Sering Diajukan",
+	"scratchwikiskin-about": "Tentang",
+	"scratchwikiskin-about-text": "Tentang Scratch",
+	"scratchwikiskin-helpthewiki": "Bantu wiki ini",
+	"scratchwikiskin-madeforscratchers": "Dibuat oleh Scratcher, untuk Scratcher.",
+	"scratchwikiskin-becomeacontributor-page": "Scratch-Indo-Wiki:Bergabung",
+	"scratchwikiskin-learnaboutjoining": "Bergabung menjadi kontributor!",
+	"scratchwikiskin-seeportal": "Portal Komunitas",
+	"scratchwikiskin-footer-about-title": "Tentang",
+	"scratchwikiskin-footer-about": "Tentang Scratch",
+	"scratchwikiskin-footer-parents": "Untuk Orangtua",
+	"scratchwikiskin-footer-educators": "Untuk Pendidik",
+	"scratchwikiskin-footer-developers": "Untuk Pengembang",
+	"scratchwikiskin-footer-credits": "Scratch Team",
+	"scratchwikiskin-footer-jobs": "Lowongan",
+	"scratchwikiskin-footer-press": "Pers",
+	"scratchwikiskin-footer-community-title": "Komunitas",
+	"scratchwikiskin-footer-cgs": "Pedoman Komunitas",
+	"scratchwikiskin-footer-discuss": "Forum Pembahasan",
+	"scratchwikiskin-footer-wiki": "Scratch Wiki",
+	"scratchwikiskin-footer-stats": "Statistik",
+	"scratchwikiskin-footer-cp": "Portak Komunitas",
+	"scratchwikiskin-footer-support-title": "Dukungan",
+	"scratchwikiskin-footer-help-page-url": "Bantuan:Isi",
+	"scratchwikiskin-footer-help": "Bantuan Untuk Penyunting",
+	"scratchwikiskin-footer-tips": "Tips",
+	"scratchwikiskin-footer-faq": "Pertanyaan yang Sering Diajukan",
+	"scratchwikiskin-footer-offline": "Editor Luring",
+	"scratchwikiskin-footer-store": "Toko Scratch",
+	"scratchwikiskin-footer-donate": "Donasi",
+	"scratchwikiskin-footer-legal-title": "Legal",
+	"scratchwikiskin-footer-privacy": "Kebijakan Privasi",
+	"scratchwikiskin-footer-family-title": "Keluarga Scratch",
```

Comment:
It turns out that the QQQ file is missing the "dmca" string. See [this line](https://github.com/InternationalScratchWiki/ScratchWikiSkin2/blob/master/i18n/en.json#L54) for what it is in English. Could you add it too?